component-catalog/src/Examples/Table.elm

```diff
@@ -135,6 +134,80 @@ example =
+            , Heading.h3 [ Heading.plaintext "With placeholderColumn", Heading.css [ Css.marginTop (Css.px 30) ] ]
+            , Table.view []
+                [ Table.rowHeader
+                    { header = text "User ID"
+                    , view = text << String.fromInt << .userId
+                    , width = Css.px 80
+                    , cellStyles = always []
+                    , sort = Nothing
+                    }
+                , Table.string
+                    { header = "First Name"
+                    , value = .firstName
+                    , width = Css.calc (Css.pct 50) Css.minus (Css.px 250)
+                    , cellStyles = always []
+                    , sort = Nothing
+                    }
+                , Table.string
+                    { header = "Last Name"
+                    , value = .lastName
+                    , width = Css.calc (Css.pct 50) Css.minus (Css.px 250)
+                    , cellStyles = always []
+                    , sort = Nothing
+                    }
+                , Table.placeholderColumn
+                    { width = Css.px 125
+                    }
+                , Table.placeholderColumn
+                    { width = Css.px 250
+                    }
+                ]
+                data
+            , Heading.h3 [ Heading.plaintext "Without placeholderColumn", Heading.css [ Css.marginTop (Css.px 30) ] ]
+            , Table.view []
+                [ Table.rowHeader
+                    { header = text "User ID"
+                    , view = text << String.fromInt << .userId
+                    , width = Css.px 80
+                    , cellStyles = always []
+                    , sort = Nothing
+                    }
+                , Table.string
+                    { header = "First Name"
+                    , value = .firstName
+                    , width = Css.calc (Css.pct 50) Css.minus (Css.px 250)
+                    , cellStyles = always []
+                    , sort = Nothing
+                    }
+                , Table.string
+                    { header = "Last Name"
+                    , value = .lastName
+                    , width = Css.calc (Css.pct 50) Css.minus (Css.px 250)
+                    , cellStyles = always []
+                    , sort = Nothing
+                    }
+                , Table.string
+                    { header = "Submitted"
+                    , value = .submitted >> String.fromInt
+                    , width = Css.px 125
+                    , cellStyles = always [ Css.textAlign Css.center ]
+                    , sort = Nothing
+                    }
+                , Table.custom
+                    { header = text "Actions"
+                    , width = Css.px 250
+                    , view = \_ -> Button.button "Action" [ Button.small ]
+                    , cellStyles = always []
+                    , sort = Nothing
+                    }
+                ]
+                data
```

Comment:
What is the purpose of this last table? At a glance it doesn't seem to be any different from the "basic" example at the top, and it seems strange to have a "Without placeholderColumn" example in a section called "Using placeholderColumn [...]".

I think that adding the new "With placeholderColumn" table should be enough. It shows the new behavior clearly, and it also happens to align with the columns from the basic example (that could change in the future, of course, but I think that adding a third table just to show that they align isn't worth the noise).

Reply:
Yeah, make sense, it was more to showcase when to use it.